docs/blueprints/EP031.rst

```diff
@@ -0,0 +1,312 @@
+===============================================
+The NoviWare's INT implementation requires `send_report` action to be executed in Table 0. `send_report` is executed with higher priority than other INT actions, which means adding INT metadata at the INT Sink has to be performed before `send_report` which is not possible on the same set of flow actions. To add INT metadata at the INT Sink, the packets have to be re-injected into the pipeline using external connections via physical loops.
+
+To illustrate the challenge, consider an EVC terminating on INT Hop Z on port 23. The user packet with INT metadata comes from port 11. **mef_eline** would create the following flows (for simplicity, just one direction is presented):
+
+  0. **met_eline** default behavior:
+
+    - match:
+
+      - priority: DEFAULT (DEFAULT FOR EVPL)
+      - match: <in_port=11,vlan_vid=20>
+
+    - instruction:
+
+      - action [set_queue, pop_vlan, etc.]
+      - action: output to port 23.
+
+To enable INT, first a physical loop has to be deployed. For this example, on INT Hop Z, port 1 is connected to port 2 by a physical fiber patch cord (done during commissioning). Then, the following flows need to be ADDED to the pipeline:
+
+  1. Adding INT metadata:
+
+    - match:
+
+      - table 0
+
+        - priority: Higher (mef_eline + 100)
+        - match: <in_port=\ **11**,vlan_vid=20,eth_type=0x800,ip_proto=6>  # TCP
+        - instrutions:
+
+          - action: add_int_metadata
+          - action: output port **1** (loop)
+
+        - priority: Higher (mef_eline + 100)
+        - match: <in_port=\ **11**,vlan_vid=20,eth_type=0x800,ip_proto=17>. # UDP
+        - instrutions:
+
+          - action: add_int_metadata
+          - action: output port **1** (loop)
+
+  2. Send Report and pop INT data (traffic is coming from port 2 that's the loop with port 1). Only INT data gets into the loop.
+
+    - match:
+
+      - table 0
+
+        - priority: Higher (mef_eline + 100)
+        - match: <in_port=\ **2**,vlan_vid=20>
+        - instrutions:
+
+          - action: send_report
+          - action go to table 2
+
+
+      - table 2
+
+        - priority: Higher (mef_eline + 100)
+        - match: <in_port=\ **2**,vlan_vid=20>
+        - instrutions:
+
+          - action: pop_int
+          - action [set_queue, pop_vlan, etc.]
+          - action: output port **23** (original port)
+
+
+IV. How to enable INT for EVCs
+==============================
+
+The goal for the **telemetry** app is to enable telemetry for ALL EVCs. However, it must support enabling and disabling telemetry for a single EVC or ALL EVCs. This is the approach:
+
+  1 . The **telemetry** napp will start operating once **mef_eline** is loaded and EVCs and their flows are pushed to the data plane.
+
+  2. **telemetry** will listen for events *kytos/mef_eline.(redeployed_link_(up|down)|deployed)* and *kytos.mef_eline.created* issued by **mef_eline**.
+
+  3. For each EVC identified, **telemetry** will
```

Comment:
We'll need to map a new arg url filter `cookie` on `v2/stored_flows`. `flow.cookie` is already indexed but not exposed to to be queried via endpoints.

Reply:
Mapped here https://github.com/kytos-ng/flow_manager/issues/124